ci/docker-entrypoint.sh

```diff
@@ -1,14 +1,23 @@
 #!/bin/bash
+
 set -e
 
 rm -rf build
 mkdir -p build
 cd build || exit 1
 
-mkdir -p /usr/lib/sysimage/tdnf
-cmake -DHISTORY_DB_DIR=/usr/lib/sysimage/tdnf .. && make -j32 && make python -j32 && make check -j32 || exit
+JOBS=$(( $(nproc) / 2 ))
```

Comment:
Good practice. Better to leave half the cores for other components usage.

Reply:
It will fail if we have just CPU:
```
$ echo $(( 1 / 2 ))
0
```
And make does not like it:
```
# make -j0
make: the '-j' option requires a positive integer argument
```

Reply:
Using `$(( ($(nproc)+1) / 2 ))` should fix it.

Reply:
Correct, good catch. I thought of this scenario once and forgot to make the changes. Thanks.